src/annotator/sidebar.js

```diff
@@ -171,6 +176,7 @@ export default class Sidebar extends Guest {
   destroy() {
     this._hammerManager?.destroy();
     this.frame?.remove();
+    this.hypothesisSidebar?.remove();
```

Comment:
If `this.hypothesisSidebar` is set then it contains `this.frame` doesn't it? If so then `this.frame` doesn't need to be removed as well. In other words, I think this logic have the same effect:

```js
if (this.hypothesisSidebar) {
  this.hypothesisSidebar.remove();
} else {
  this.frame?.remove();
}
```

I'm OK with keeping the existing logic as is, I'm just noting that we could simplify it when the flag is removed.

Reply:
Yes, it is true, your suggested logic has the same effect.

I will simplify once we remove the flag.

Reply:
Interestingly, the `this.externalFrame` is never removed. I don't know if that was intentional or not.